packages/mc-html-template/html-docs/application.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
```

Comment:
Moving this in to a proper HTML file makes it a bit "easier" to maintain and read (syntax highlighting) instead of being a huge block of string.